eos-upgrade-eos2-to-eos3

```diff
@@ -167,8 +167,8 @@ rm -f /var/lib/chromium-plugins-extra/eos-chrome-plugin-updater/VERSION.txt
 # we can get a complete list of printers regardless of their status
 # as well as to ensure that the configuration files handled by CUPS
 # e.g. /etc/cups/printers.conf) are correctly updated on restart.
-systemctl stop cups 2> /dev/null
-printers=$(lpstat -p 2> /dev/null | grep ^printer | cut -d ' ' -f 2)
+systemctl stop cups
+printers=$(lpstat -p | grep ^printer | cut -d ' ' -f 2)
```

Comment:
This change has no effect. The script does not set `-o pipefail`. By default, the exit status of a pipeline is the exit status of its rightmost command:

```console
$ false | true; echo $?
0
$ true | false; echo $?
1
```

So the existing command will only fail if `cut` fails, which .. presumably it does not!

Reply:
Talking to Will we actually think we should still do this change, if anything to see the error output when it fails, as that won't get redirected into the pipe